docs/decisions/0022-settings-simplification.rst

```diff
@@ -0,0 +1,327 @@
+      which is not documented.
+
+* ``cms/envs/tutor/production.py``...
+
+  * is generated by Tutor from the template
+    ``tutor/templates/apps/openedx/settings/cms/production.py``,
+
+    * which derives
+      ``tutor/templates/apps/openedx/settings/partials/common_cms.py``,
+
+      * which derives
+        ``tutor/templates/apps/openedx/settings/partials/common_all.py``;
+
+    * and uses templates vars from Tutor configuration (``config.yml``),
+
+    * and invokes hooks from any enable Tutor plugins;
+
+  * it imports ``cms/envs/production.py``,
+
+    * it imports ``cms/envs/common.py``, which sets production-inappropriate
+      defaults,
+
+      * and which imports ``lms/envs/common.py``, which also sets
+        production-inappropriate defaults;
+
+    * it sets more defaults, some of the edX.org-specific;
+
+    * it loads ``/openedx/config/cms.yml``...
+
+      * which is generated by Tutor from template
+        ``tutor/templates/apps/openedx/config/cms.env.yml``
+
+        * which derives
+          ``tutor/templates/apps/openedx/config/partials/auth.yml``;
+
+    * it reverts some of ``/openedx/config/cms.yml`` with new "defaults";
+
+    * and it uses certain values ``/openedx/config/cms.yml`` to conditionally
+      override more settings and update certain dictionary settings, in a way
+      which is not documented.
+
+This is very difficult to reason about. Configuration complexity is frequently
+cited as a chief area of pain for Open edX developers and operators.
+Discussions in the Named Release Planning and Build-Test-Release Working Groups
+frequently are encumbered with confusion and uncertainty of what the default
+settings are in edx-platform, how they differ from Tutor's default settings,
+what settings can be overriden, and how to do so. Only a minority of developers
+and operators fully understand the configuration logic described above
+end-to-end; even for those that do, following this override chain for any given
+Django setting is time-consuming and error-prone. CAT-1 bugs and high-severity
+security vulnerabilities have arisen due to misunderstanding of how
+edx-platform Django settings are rendered.
+
+Developers are frequently instructed that if they need to override a Django
+setting, the preferred way to do so is to "make a Tutor plugin". This is a
+large amount of prior knowledge, boilerplate, and indirection, all required
+to simply do something which Django provides out-of-the-box via a custom
+``DJANGO_SETTINGS_MODULE``.
+
+Finally, it is worth nothing that all the complexity and toil exists alongside
+other edx-platform configuration methods, such as Waffle, configuration models,
+site configuration, XBlock configuration, and entry points. Those configuration
+pathways are outside of the scope of this ADR, but are mentioned to demonstrate
+the distressing level of complexity that developers and operators face when
+working with the platform.
+
+Decision
```

Comment:
Consider adding a sentence or two in this Decision section about the decision to adjust OEP-45 in a separate effort to better align with the decisions noted in this ADR. You can also reference the Consequences section for more details.

Reply:
I've merged Decision & Consequences together, and tried to better weave the OEP-45 relationship into the combined section.